runtime/commonTest/src/kotlinx/serialization/protobuf/ProtobufReaderTest.kt

```diff
@@ -59,4 +59,9 @@ class ProtobufReaderTest {
         if (isNative()) return // todo: support update on Native
         ProtoBuf.loads(TestIntWithList.serializer(), "500308960150045005") shouldBe TestIntWithList(150, listOf(3, 4, 5))
     }
+
+    @Test
+    fun readObjectWithNullableMissingFields() {
```

Comment:
There are no new tests that will verify that nullable fields with `kind != Kind.CLASS` are indeed prohibited.